docs/tutorials/http-serving-tutorial.md

```diff
@@ -84,15 +124,23 @@ interface HTTPRequest {
 
 ### Outputs and HTTP Responses
 
-By default, if a function invoked via HTTP request returns successfully, its return value is sent in the HTTP response body with status code `200` (or `204` if nothing is returned).
+If a function invoked via HTTP request returns successfully, its return value is sent in the HTTP response body with status code `200` (or `204` if nothing is returned).
+
 If the function throws an exception, the error message is sent in the response body with a `400` or `500` status code.
-If the error contains a `status` field, the handler uses that status code instead.
+If the error contains a `status` field, the response uses that status code instead.
```

Comment:
We can mention `DBOSResponseError` here -- it contains a `status` field.

Reply:
Separate PR probably